lib/ukrust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "ukrust"
+version = "0.1.0"
+edition = "2018"
```

Comment:
We should probably also add a license here

Reply:
Makes sense!